features/storeHealthCertificate.feature

```diff
@@ -0,0 +1,45 @@
+@RepositoryService
```

Comment:
I might also tag this feature with @StoreHealthCertificate, and change the success scenario to @StoreHealthCertificate-valid. This way the user can run all StoreHealthCertificate tests at once using the tag.

Reply:
You can tag the entire file. With both @RepositoryService and @StoreHealthCertificate. Gherkin allows more than one tag. Since defaults relies on @RepositoryService, that is also required to set the endpoint.